Gemfile

```diff
@@ -23,7 +23,7 @@ gem 'puma', '~> 3.11'
 gem 'bootsnap', '>= 1.4.2', require: false
 
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
-# gem 'rack-cors'
+gem 'rack-cors'
```

Comment:
?

Reply:
Added origin to CORS config to be env var

Reply:
This is the middleware to allow us to specify appropriate headers for Access-Control-Allow-Origin header and which HTTP methods are exposed.